### PR TITLE
*: remove until timestamp syntax in flashback statement since we not support this

### PR DESCRIPTION
**Pull request by crazycs520**
### What problem does this PR solve? 

Related parser PR: https://github.com/pingcap/parser/pull/733, Should merge with parser at the same time.

PR https://github.com/pingcap/tidb/pull/13109/ add the `flashback` syntax. But it was hard to use with the `until timestamp` statement. And currently, we can't support `flashback` to any timestamp, even in the GC safe time. 

`Flashback` is only supported to flashback table to the last drop/truncate time, and the time time should within the `tikv_gc_safe_point` time.

After think over and talk with PM, we decided to remove the `until timestamp` statement in the `flashback` syntax. Now the `flashback` syntax is like below:

```SQL
FLASHBACK TABLE table_name [TO] [other_name]
```

eg:

```sql
test> create table t (a int key,b int);
test> insert into t values (1,1);
test> truncate table t; -- truncate the table;
test> select * from t;
+---+---+
| a | b |
+---+---+
test> flashback table t to t1;  -- flashback table t to t1;
test> select * from t1;
+---+---+
| a | b |
+---+---+
| 1 | 1 |
+---+---+
test> insert into t1 values (2,2)
test> select * from t1;
+---+---+
| a | b |
+---+---+
| 1 | 1 |
| 2 | 2 |
+---+---+
test> select * from t;
+---+---+
| a | b |
+---+---+
```

### Check List 

Tests 

 - Unit test

Code changes

 - Has exported function/method change

Side effects

Related changes

Release note

### CODE REVIEW

**Comment by crazycs520** [comment]
@djshow832 @AilinKid PTAL

**Comment by crazycs520** [comment]
CI was failed in `go.mod`, after [the parser PR_733](https://github.com/pingcap/parser/pull/733) merged and update the `go.mod`, CI will happy.

File: executor/executor_test.go [hunk 1]
```diff
@@ -4652,22 +4652,17 @@ func (s *testRecoverTable) TestFlashbackTable(c *C) {
 	tk.MustExec("insert into t_flashback values (1),(2),(3)")
 	tk.MustExec("drop table t_flashback")
 
-	// Test flash table with wrong time.
-	_, err = tk.Exec(fmt.Sprintf("flashback table t_flashback until timestamp '%v'", time.Now().String()))
-	c.Assert(err.Error(), Equals, "Can't find dropped table: t_flashback in ddl history jobs")
-
```
Comment (djshow832) [inline executor/executor_test.go]: Why not check wrong name?

Comment (crazycs520) [reply executor/executor_test.go]: Great, done.

**Pull request merged**